prow/plugins/config.go

```diff
@@ -477,6 +478,20 @@ type Welcome struct {
 	MessageTemplate string `json:"message_template,omitempty"`
 }
 
+// Dco is config for the DCO plugin.
```

Comment:
Please define DCO somewhere

Reply:
I added a link there. It is also defined like that in the `dco` package.